apps/pie-docs/src/foundations/colour/overview.md

```diff
@@ -28,12 +28,12 @@ caption: "Some of the meanings we have determined for our ‘Supportive’ colou
 
 ### Colours follow accessibility guidelines
 
-The colour system is designed within the HSLuv colour space to generate themes that meet WCAG 2.1 compliant contrast ratios. This makes things easier to find, identify, and interact with. It also makes the whole experience more accessible for merchants who are color blind or who have low vision. However you should never convey information using color alone.
+The colour system is designed within the HSLuv colour space to generate themes that meet WCAG 2.1 compliant contrast ratios. This makes interface elements easier to find, identify, and interact with. It also makes the whole experience more accessible for users who are colour-blind or who have low vision. However you should never convey information using colour alone.
```

Comment:
Adding a link here because I hadn't heard of HSLuv and thought others might appeciate a reference.

```suggestion
The colour system is designed within the [HSLuv](https://www.hsluv.org) colour space to generate themes that meet WCAG 2.1 compliant contrast ratios. This makes interface elements easier to find, identify, and interact with. It also makes the whole experience more accessible for users who are colour-blind or who have low vision. However, you should never convey information using colour alone.
```

Reply:
Good shout, thanks! I'll make the appropriate changes :)

Reply:
👍